themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -65,6 +65,9 @@
               <?=$this->icon('place-recall', 'icon-link__icon') ?>
               <span class="icon-link__label"><?=$this->transEsc($check ? 'Check Recall' : 'Recall This')?></span>
             </a>
+	  <?php endif; ?>
+	  <?php if ($holding['in_transit']): ?>
```

Comment:
Since in_transit is an optional field, this needs null coalescing (e.g `<?php if ($holding['in_transit'] ?? false): ?>`).